client/apps/edit/components/admin/index.coffee

```diff
@@ -308,17 +308,21 @@ module.exports = class EditAdmin extends Backbone.View
       featuredListTemplate list: @article.featuredList('Artworks')
     )
 
+  renderInputs: (src) ->
+    $('.edit-email-large-image-url input').val(
+      crop(src, { width: 1280, height: 960 } ))
```

Comment:
TBD what will replace it, but it does seem so far we're leaning towards going all in on [Gemini](http://github.com/artsy/gemini) or potentially using [this Node library](https://github.com/lovell/sharp)